specification/under_development/profiles/Base_Profile.md

```diff
@@ -0,0 +1,484 @@
+# Base Profile
+
+This profile defines a subset of the QIR specification to support a coherent set
+of functionalities and capabilities that might be offered by a quantum backend.
+Like all profile specifications, this document is primarily intended for
+[compiler backend](https://en.wikipedia.org/wiki/Compiler#Back_end) authors as
+well as contributors to the [targeting
+stage](../Compilation_And_Targeting.md#targeting) of the QIR compiler.
+
+The Base Profile specifies the minimal requirements for executing a quantum
+program. Specifically, to execute a Base Profile compliant program, a backend
+needs to support the following:
+
+1. It can execute a sequence of quantum instructions that transform the quantum
+   state.
+2. It supports measuring the state of each qubit at the end of the program.
+3. It produces one of the specified [output schemas](../output_schemas/).
+
+These functionalities are both necessary and sufficient for computations that
```

Comment:
"both" but there are now three items.

Reply:
The both was intended to refer to "necessary" and "sufficient", but I just remove it; that is clearer in any case.